### ISSUE TITLE
[New Content]: <blog pages need a publish date>

### ISSUE THREAD

**Issue by MarcelCloud** [scroll-tech/scroll-documentation#172]
### What would you like us to add?

Hi, going through the scroll [blog ](https://scroll.io/blog/architecture) and I'd suggest the team add a publish date. 

### How important it is?

It's important to outline how far scroll has come and a content timeline pointing to that would do a whole lot of good.

**Comment by dghelm** [comment]
I agree that this is a great suggestion, as it will help contextualize potential stale documents. We'll explore adding this in future site updates.
